pkg/cloudprovider/provider/gce/service.go

```diff
@@ -0,0 +1,129 @@
+//
+// Google Cloud Provider for the Machine Controller
+//
+
+package gce
+
+//-----
+// Imports
+//-----
+
+import (
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2"
+	"google.golang.org/api/compute/v1"
+)
+
+//-----
+// Constants
+//-----
+
+const (
+	// Timeouts.
+	timeoutNormal = 10 * time.Minute
```

Comment:
Having a global timeout of 10 minutes does not really look right for all API calls we do

Reply:
Changed to interval of 5 seconds and timeout of 5 minutes as seen in other project.